pkg/watermark/processor/processor_manager_test.go

```diff
@@ -213,7 +213,7 @@ func TestProcessorManagerWatchForMap(t *testing.T) {
 		err = otStore.PutKV(ctx, "p1", otValueByte)
 		assert.NoError(t, err)
 	}
-	for processorManager.GetProcessor("p1").GetOffsetTimeline().GetHeadOffset() == -1 || processorManager.GetProcessor("p2").GetOffsetTimeline().GetHeadOffset() == -1 {
+	for processorManager.GetProcessor("p1").GetOffsetTimeline().GetHeadOffset() != 115 || processorManager.GetProcessor("p2").GetOffsetTimeline().GetHeadOffset() != 115 {
```

Comment:
why did we have to change this in the test case?

Reply:
It was flaky.